src/graph/validator/MatchValidator.cpp

```diff
@@ -1292,19 +1292,25 @@ Status MatchValidator::validatePathInWhere(
 }
 
 /*static*/ Status MatchValidator::buildRollUpPathInfo(const MatchPath *path, Path &pathInfo) {
-  DCHECK(!DCHECK_NOTNULL(path->alias())->empty());
   for (const auto &node : path->nodes()) {
     // The inner variable of expression will be replaced by anno variable
     if (!node->alias().empty() && node->alias()[0] != '_') {
       pathInfo.compareVariables.emplace_back(node->alias());
     }
   }
   for (const auto &edge : path->edges()) {
-    if (edge->alias()[0] != '_') {
-      pathInfo.compareVariables.emplace_back(edge->alias());
+    const auto &edgeAlias = edge->alias();
+    if (!edgeAlias.empty() && edgeAlias.front() != '_') {
```

Comment:
We should try to avoid this kind of code.
Maybe `edge->isAnonymous()` is more appropriate.

Reply:
fixed